PyInstaller/building/makespec.py

```diff
@@ -319,13 +319,11 @@ def main(scripts, name=None, onefile=None,
         # version is >= 2.4.
         try:
             import Crypto
-
-            pycrypto_version = list(map(int, Crypto.__version__.split('.')))
-            is_version_acceptable = pycrypto_version[0] >= 2 and pycrypto_version[1] >= 4
-
+            major, minor,_,_,_ = Crypto.version_info
+            temp_version = major*10+minor
+            is_version_acceptable = 24 <= temp_version
```

Comment:
Why not just
 `is_version_acceptable = (major > 2 and minor > 4)`
or
`is_version_acceptable = ((major, minor) > (2, 4))`
?